### ISSUE TITLE
Kindle Android - book unavailable in this version of the application

### ISSUE THREAD

**Issue by BRASPO22** [apprenticeharper/DeDRM_tools#2316]
The following message appears when I click on the exclamation mark that appears below the ebook image:
Book unavailable in this version of the application
You are using an unsupported version of Kindle for Android.
To read this book, update to the latest version on your preferred Android app store.

Kindle Android is asking to update or read the ebook on another device. Currently using a Samsung J2 Prime Android 6 cell phone. I tested it on an Android 4.1.2 cell phone and the problem is the same. Amazon won't let you download and the prc doesn't appear. Ebooks 2024. The ebooks that you can still download from other years caliber cannot break the drm of the prc file. Amazon certainly made some changes there and at least for me the DRM breakage fell through using Android. Is there currently another way to break the drm other than buying the ebook? Thanks in advance.

**Comment by ElleKayEm** [comment]
I'm still able to use Kindle for Android 4.16.0.75 on BlueStacks.  There's also Kindle for PC/Mac with DeDRM 10.0.9 and KFX Input plugins.

**Comment by BRASPO22** [comment]
I updated Caliber recently and I believe this is what caused the drm not breaking issue.
Now I deleted the 2 KFX's plugins (input and output) and the ddrm 10.0.9 plugin and installed it again and it worked.
I have to remember that every time I update Caliber I need to uninstall the plugins again even though they are installed.
Now my only problem is the releases from May 10th onwards that no longer download on my phone.
Thank you so much for always helping me when I need it.
It was really worth it.
♡♡♡♡♡♡♡♡♡
